src/store/mutations/mutations-model.ts

```diff
@@ -207,6 +207,7 @@ export const mutateStaffPayment = (state: StateIF, staffPayment: StaffPaymentIF)
 
 export const mutateStaffPaymentValidity = (state: StateIF, validity: boolean) => {
   state.stateModel.staffPaymentStep.valid = validity
+  state.stateModel.newAlteration.validFlags.isValidStaffPayment = validity
```

Comment:
Don't change this right now... This comment is for all devs: this is a bit of a code smell...

I think we need to figure out our architecture for data that applies to different filings (like here - Corrections + Alterations). This is one approach but it results in duplicate code and won't scale well. I think a better approach would be to generalize data like this that is used in different filings. What do you guys think?

(I'm OK with this for now and we can deal with this if/when we add another filing to this project, but it would be nice to have an idea/plan what to do when we get there.)